pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -33,16 +33,27 @@ import (
 )
 
 const (
-	OrchestrationStatusStarting      = "starting"
+	// OrchestrationStatusStarting denotes the OSD provisioning is beginning.
+	OrchestrationStatusStarting = "starting"
+	// OrchestrationStatusComputingDiff denotes the OSD provisioning is scanning for disks that can
+	// be provisioned.
 	OrchestrationStatusComputingDiff = "computingDiff"
+	// OrchestrationStatusOrchestrating denotes the OSD provisioning has begun and is running.
 	OrchestrationStatusOrchestrating = "orchestrating"
-	OrchestrationStatusCompleted     = "completed"
-	OrchestrationStatusFailed        = "failed"
-	orchestrationStatusMapName       = "rook-ceph-osd-%s-status"
-	orchestrationStatusKey           = "status"
-	provisioningLabelKey             = "provisioning"
-	nodeLabelKey                     = "node"
-	completeProvisionTimeout         = 20
+	// OrchestrationStatusCompleted denotes the OSD provisioning has completed. This does not imply
+	// the provisioning completed successfully in whole or in part.
+	OrchestrationStatusCompleted = "completed"
+	// OrchestrationStatusPreexisting denotes the OSD provisioning was not started because an OSD
+	// has already been provisioned and started.
+	OrchestrationStatusPreexisting = "preexisting"
+	// OrchestrationStatusFailed denotes the OSD provisioning has failed.
+	OrchestrationStatusFailed = "failed"
```

Comment:
Currently, this PR doesn't actually change behavior (still working on that), but it does ad an extra status. I wanted to CI still run green before going too deep. How do you feel about the updated descriptions of the statuses?